blog/_posts/2022-01-03-arm-wrestling-in-your-dev-env.md

```diff
@@ -0,0 +1,103 @@
+---
+slug: "arm-wrestling-in-dev"
+date: 2022-01-04
+author: nick
+layout: blog
+title: "ARM Wrestling the Right Processor in Dev"
```

Comment:
tbh i think this needs a better title. 

Maybe 2022 is the Year You Use ARM in Dev
?

Reply:
FWIW, I like the pun, but it loses a bit with "the right processor".

Some alternative suggestions:
- ARM Wrestling your Dev Environment
- Fully ARM your Dev Environment
- Working in Dev without losing an ARM and a leg

Or no pun:
- Making ARM chips work with Tilt/Dev
- ARM/M1 + Tilt = ❤️

Reply:
done!